### ISSUE TITLE
Fixing accessibility issues in your themes

### ISSUE THREAD

**Issue by lubiah** [spences10/scottspence.com#429]
Hi, your site really looks great and I love the design but the texts are hard to read. When run on an accessibility check, it fails most of the color contrast checks

**Comment by spences10** [comment]
Hi @kudadam,

Great shout! I've seen _some_ contrast issues, not _most_  like you say. The themes are from [daisyUI] I'm sure there's probably an issue in there for the contrast a11y checks.

If not you can suggest it there: https://github.com/saadeghi/daisyui

Thanks, closing this now 😊 

[daisyui]: https://daisyui.com
